src/PDS4_PROC_IngestLDD.xml

```diff
@@ -33,12 +33,31 @@
         - Reworded `Software` description
         - Removed child checking DD_Rules
         - Added new rules for reference_type and internal_reference_type checks
-    
+
+        1.4.0.0 R.Joyner
+        - Update to 1.21.0.0 / 1.L.0.0
+        - Added software_availability as new attribute
+
     </comment>
     <last_modification_date_time>2022-06-08T20:00:00Z</last_modification_date_time>
     
     <!-- Attributes START -->
 
+    <DD_Attribute>
+        <name>software_availability</name>
+        <version_id>1.0</version_id>
+        <local_identifier>software_availability</local_identifier>
+        <nillable_flag>false</nillable_flag>
+        <submitter_name>Ronald Joyner</submitter_name>
+        <definition>The software_availability attribute refers to the extent
+            to which a software application is accessible and operational when 
+            needed and is an aspect of software quality.</definition>
+        <DD_Value_Domain>
```

Comment:
@rsjoyner is there a reason we did not use the enumeration from here? https://github.com/pds-data-dictionaries/ldd-proc/pull/26/files#diff-d68a0708ee95b43a062248a4a123679ccbbd19c6aebd8dc43606af89a37a15e6R191

I think those definitions are helpful context for a user (and make it into the IM) versus just having the rule check.